python/spacewalk/common/repo.py

```diff
@@ -466,20 +476,17 @@ def _get_release_index_from_file(self) -> typing.Dict[str, "DpkgRepo.ReleaseEntr
                     if self.gpg_verify and not self._has_valid_gpg_signature(
                         local_path
                     ):
-                        logging.error(
-                            # pylint: disable-next=logging-format-interpolation,consider-using-f-string
-                            "GPG verification failed: '{}'. \
-                                           Raising GeneralRepoException.".format(
-                                release_file
-                            )
+                        logger.error(
+                            "%s: %s. Raising GeneralRepoException.",
+                            DpkgRepo.GPG_VERIFICATION_FAILED,
+                            release_file,
```

Comment:
There are other instances of this as well. I'll fix it and force push with a clean history

Reply:
Thanks!